.gitattributes

```diff
@@ -0,0 +1,5 @@
+# Set the default behaviour, in case poeple don't have core.autocrlf set
+* text=auto
+
+# Always lf, this file size is checked in a test
+tests/fixtures/static/encoded.html text eol=lf
```

Comment:
And this file was getting checked out, and it's line endings were changed, however a test was counting bytes inside the file.

So here we explicitly set the `eol` to linefeed so that git doesn't change the file size accidentally.